### DISCUSSION TITLE
Allow validating candidate configuration

### DISCUSSION THREAD

**Discussion by Nothing4You** [PaloAltoNetworks/pan-os-ansible#211] [Ideas]
useful for example when generating an xml configuration and uploading it

**Comment by mrichardson03** [comment]
Validation is an op command, and can be done with 'panos_op'.

**Reply by Nothing4You** [reply]
unfortunately that only returns a job id from an enqueued validation job.
unlike `panos_commit_firewall` there is no `sync` parameter for `panos_op`.
this seems to be a limitation of `pan-os-python` python though, so i guess it should be requested there instead?

it could probably still be done without explicit support for `sync` in `panos_op` in `pan-os-python` though by calling [`syncjob`](https://github.com/PaloAltoNetworks/pan-os-python/blob/1a53e9b92267fce25fbb1ce0760e3d90a9055a10/panos/base.py#L4668) directly.
